.gitignore

```diff
@@ -8,5 +8,5 @@ tmp
 docker/conf/certificates/*.crt
 *.log
 # any presto binary
-presto*
+#presto*
```

Comment:
It was only because it ignored the new folder starting with *presto* 😅 

Is it safe to remove it? I don't know potentially were the presto binary comes from ...

Reply:
I tested with `presto` binary(I have no plan to continue playing with presto binary). I mean here, why you put this line under comment?

Reply:
Like I said, because the folder `presto_cluster` got ignored, but I didn't know it was safe to delete or not

Reply:
It's deleted btw 😄